dashboard/test/ui/utils/selenium_browser.rb

```diff
@@ -41,7 +41,7 @@ def create_response(code, body, content_type)
       if (msg = exception.message.match(/unexpected response, code=(?<code>\d+).*\n(?<error>.*)/))
         error = msg[:error]
         error = JSON.parse(error)['value']['error'] rescue error
-        exception.message.replace("Error #{msg[:code]}: #{error}")
+        raise exception, "Error #{msg[:code]}: #{error}", exception.backtrace
```

Comment:
I definitely like this approach better than mutating the exception.

Reply:
Likewise! Shout out to @snickell, who actually came up with this before I copied it out of his WIP PR. I assume it was in response to the fact that the message string is now frozen internally by Selenium and throws an error when we try to modify it, but I agree that overall it's much cleaner this way. Also, I learned that you can pass the backtrace manually as an optional third argument to `Kernel#raise`!